editor.planx.uk/src/@planx/components/FileUploadAndLabel/Editor.test.tsx

```diff
@@ -41,7 +41,7 @@ describe("FileUploadAndLabel - Editor Modal", () => {
     expect(screen.getAllByText("File")).toHaveLength(1);
   });
 
-  it.skip("allows an Editor to add multiple rules", async () => {
```

Comment:
Nice catch! Worth us probably checking out all existing `skips()` once this vite + vitest migration is done and maybe opening GitHub issues for them.